app/master/time_based_atom_grouper.py

```diff
@@ -103,28 +103,27 @@ def groupings(self):
     def _coalesce_new_atoms_with_historic_times(self, new_atoms, old_atoms_with_times, project_directory):
         """
         Combine the historic timing data from previous atoms (for the same job) with the new atoms generated for this
-        job.
+        job. This will update the expected_time fields for the atom objects in new_atoms.
```

Comment:
"Set the expected run time (new_atom.expected_time)  of each atom in new_atoms using historic timing data."